schemas/felles/frittstaende-ord/fraUtlandetNorge.ts

```diff
@@ -0,0 +1,31 @@
+import {
+  CustomSanityTyper,
+  DokumentBase,
+  DokumentNavn,
+  dokumentTittel,
+  SanityTyper,
+} from '../../typer';
+
+const fraUtlandetNorge: DokumentBase = {
```

Comment:
Hmm litt usikker på denne sammenslåingen, for da er den jo veldig sterkt knyttet til flettefelt igjen når det egentlig bare er frittstående ord 🤔 at det hadde kanskje vært like greit å hente det ut på frittståendeOrd.norge og frittståendeOrd.utlandet

Reply:
og at "fra" ordet holder seg til det andre dokumentet